packages/front/src/components/Claim.jsx

```diff
@@ -1,18 +1,18 @@
 import React, { Component } from 'react'
 import {BrowserRouter, Switch, Route} from 'react-router-dom'
-import SpeechView from './SpeechView'
-import SpeechCreate from './SpeechCreate'
+import ClaimView from './ClaimView'
+import ClaimCreate from './ClaimCreate'
 
-const Speech = ({match}) => {
-    console.log("Speech", match.params.id);
+const Claim = ({match}) => {
+    console.log("Claim", match.params.id);
```

Comment:
Can remove 'console.log()'. This is not necessary.

Reply:
Agreed. The next patch will have this removed.